src/views/DidEndpointsSign/DidEndpointsSign.tsx

```diff
@@ -94,9 +134,31 @@ export function DidEndpointsSign({
 
       <CopyValue value={identity.did} label="DID" className={styles.didLine} />
 
-      <p className={styles.value}>{endpoint.urls[0]}</p>
-      <p className={styles.value}>{endpoint.types[0]}</p>
-      <p className={styles.value}>{getFragment(endpoint.id)}</p>
+      {fee && (
+        <p className={styles.fee}>
+          {t('view_DidEndpointsSign_fee')}
+          <KiltAmount amount={fee} type="costs" smallDecimals />
+        </p>
+      )}
+
+      <dl className={styles.details}>
+        <div className={styles.fullWidthDetail}>
+          <dt className={styles.detailName}>
+            {t('view_DidEndpointsSign_url')}
+          </dt>
+          <dd className={styles.detailValue}>{endpoint.urls[0]}</dd>
+        </div>
+        <div className={styles.detail}>
+          <dt className={styles.detailName}>
+            {t('view_DidEndpointsSign_type')}
+          </dt>
+          <dd className={styles.detailValue}>{endpoint.types[0]}</dd>
+        </div>
+        <div className={styles.detail}>
+          <dt className={styles.detailName}>{t('view_DidEndpointsSign_id')}</dt>
+          <dd className={styles.detailValue}>{endpoint.id}</dd>
```

Comment:
Previously we were displaying only the fragment part, but now the full thing?

Reply:
Oops, good catch